### DISCUSSION TITLE
Possible to publish overpowering/overheating from Shelly device?

### DISCUSSION THREAD

**Discussion by AleXSR700** [arendst/Tasmota#16474] [Q&A]
Hello everyone,
I flashed my Shelly 1, 1PM and 2.5 with Tasmota and would like to monitor the overpowering/overheating binary_sensors. Unfortunately they are not being published by Tasmota via MQTT.
Is it possible to add these somehow? I have automations in place that will react and notify me if any device is having issues.

Thank you and have a nice weekend
Alex

**Comment by barbudor** [comment]
You can use a rule to compare temperature/power and publish on a topic of your choice
```
Rule1 On energy#power>100 do publish tele/%topic%/alarm overpower=on endon on energy#power<100 do publish tele/%topic%/alarm overpower=off endon
Rule1 1
Rule1 5
```

**Reply by sfromis** [reply]
If what you mean is the temperature threshold you can config using `SetOption42`, you could make a rule to mimic the overtemperature detection, and publish explicit info via MQTT.
